### ISSUE TITLE
Framegrabbing on non-rpi hardware

### ISSUE THREAD

**Issue by revil804** [tvdzwan/hyperion#154]
Hi all,
I have a an ODROID-U3 (running Ubuntu 14.04) that i am trying to get Hyperion frame-grabbing function on. Thus far, I have Hyperion installed (complied from src) and can successfully send the effects patterns via SPI (using android app). So the hardware interface is functional...

My issue is while in XBMC (I have the Hyperion add-on installed), when i play a sample video, it is extremely choppy playback and the LEDs (WS2801) do not respond accordingly...

Ran:
sudo hyperiond /etc/hyperion.config.json

Application build time: Aug 31 2014 15:06:32
QCoreApplication initialised
Selected configuration file: /etc/hyperion.config.json
ColorTransform 'default' => [0; 83]
Device configuration: 
{
	"colorOrder" : "rgb",
	"name" : "MyPi",
	"output" : "/dev/spidev1.0",
	"rate" : 460800,
	"type" : "ws2801"
}

Black border threshold set to 0.01 (3)
Not creating any smoothing
Effect loaded: Knight rider
Effect loaded: Blue mood blobs
Effect loaded: Cold mood blobs
Effect loaded: Full color mood blobs
Effect loaded: Green mood blobs
Effect loaded: Red mood blobs
Effect loaded: Warm mood blobs
Effect loaded: Rainbow mood
Effect loaded: Rainbow swirl fast
Effect loaded: Rainbow swirl
Effect loaded: Snake
Effect loaded: Strobe blue
Effect loaded: Strobe Raspbmc
Effect loaded: Strobe white
Initializing Python interpreter
Hyperion created and initialised
run effect Rainbow swirl fast on channel 0
Boot sequence(Rainbow swirl fast) created and started
XBMC video checker created and started
The dispmanx framegrabber can not be instantiated, becuse it has been left out from the build
Json server created and started on port 19444
Proto server created and started on port 19445
XBMC Connected
Message from XBMC: {"id":666,"jsonrpc":"2.0","result":[]}
XBMC checker: switching to OFF mode
Message from XBMC: {"id":668,"jsonrpc":"2.0","result":{"System.ScreenSaverActive":true}}
Message from XBMC: {"id":670,"jsonrpc":"2.0","result":{"version":{"major":13,"minor":2,"revision":"20140803-23f8957","tag":"beta"}}}

The thing i notice there is "The dispmanx framegrabber can not be instantiated, because it has been left out from the build"

So the framegrabber is not installed?

tl;dr: Complied Hyperion from src on non-rpi hardware. dispmanx framegrabber is not installed. How do i install this?

Thanks for any help

**Comment by pdf** [comment]
@revil804 the X11 grabber is now supported in the master branch, but performance will still be unacceptable due to use of XGetImage.  I've opened issue #282 suggesting support for XRender, which would sort this out.

**Comment by tpmodding** [comment]
@revil804 can be closed?

**Comment by revil804** [comment]
Sorry for delay. Yes issue can be closed.

**Comment by tpmodding** [comment]
could you close it pleasE? i cant...
